### PR TITLE
Create MockInterviewFormDetails.jsx

### PR DESCRIPTION
**Pull request by wmx-OwO**
image
image

### CODE REVIEW

**Comment by wmx-OwO** [comment]
image
image

**Comment by wmx-OwO** [comment]
image

File: components/MockInterviewFormDetails.jsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+              <FaTimes className="text-3xl text-acm-marine m-4 hover:cursor-pointer" />
+            </div>
+            <div className="ml-4 mb-4 mr-4">
+              <p className="font-lexend text-acm-white text-lg  font-medium m-1">
+                name
+              </p>
+              <div className="flex justify-start items-center">
+                <FaCircle className="text-acm-marine text-sm" />
+                <p className="font-lexend text-acm-white text-m ml-3 font-light mb-0">
+                  {mockInterviewFormDetails.name}
```
Comment (shahdivyank) [inline components/MockInterviewFormDetails.jsx:25]: Have the data come in from props instead

File: components/MockInterviewFormDetails.jsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import { FaCircle, FaTimes } from "react-icons/fa";
+import mockInterviewFormDetails from "./MockinterviewFormData";
+
+const MockInterviewFormDetails = () => {
```
Comment (shahdivyank) [inline components/MockInterviewFormDetails.jsx:5]: Setup the props to read the data, the mockInterviewFormDetails should be in the parent component of this component and passes in the props for it

**Review by shahdivyank — APPROVED** [review]

**Review by Arnav-Menon — APPROVED** [review]

**Pull request merged**